### ISSUE TITLE
player Hide all Manialinks with a key like F9

### ISSUE THREAD

**Issue by kremsy** [ManiaControl/ManiaControl#143]

**Comment by steeffeen** [comment]
requires https://github.com/steeffeen/FancyManiaLinks/issues/45

**Comment by kremsy** [comment]
Implemented into FML, add it in the ManiaLinkManager in case the given Manialink is an instanceof ManiaLink

furthermore, maybe create a new Method in MC to create a manialink which sets this and other defaults in future.

**Comment by axelalex2** [comment]
In theory, it is in the code: https://github.com/ManiaControl/ManiaControl/blob/master/core/Manialinks/ManialinkManager.php#L253 but as the comment says, it doesn't work. But, there is another way of implementing a similar feature, which the very new EvoSC has shown, by moving most widgets out of the screen when driving at a certain speed. I can investigate, and add this feature under a setting, which does not move the widgets out of the screen (would be a little hard to manage with all the user plugins in mind), but rather fades them in and out. I'll see how well that looks and integrates with the "Simple-UI"-style.

**Comment by kremsy** [comment]
Disabling with a key would be way better than fading out automatically. Especially for shootmania. But it would be needed to be done with maniascript to move/hide all widgets, especially the ones not player baded.

**Comment by axelalex2** [comment]
I agree, that a key would be better, but if it doesn't work, and i can't figure out why, i could try that approach, which at least would work for TrackMania, obviously not really for ShootMania. And ofc, it would be client-sided ManiaScript. Not sure, what you mean by that last sentence, tho.
